rnd/autogpt_server/autogpt_server/data/block.py

```diff
@@ -16,6 +16,13 @@
 CompletedBlockOutput = dict[str, list[Any]]  # Completed stream, collected as a dict.
 
 
+class BlockUIType(Enum):
+    STANDARD = "Standard"
```

Comment:
micro nit: Can we come up with some more clear name here than standard? It works but :(

Reply:
I had also considered Default and Normal. What do you think?

Reply:
Is there a better name for the collection that describes them? Inputs is what I’d choose but that has other meanings

Reply:
I don't understand what you are getting at.

Reply:
what would the alternatives to Default be as an example?

Reply:
what would default look like in the UI?

Reply:
How it currently is a normal node